api/server/router/system/system_routes.go

```diff
@@ -26,6 +27,7 @@ func optionsHandler(ctx context.Context, w http.ResponseWriter, r *http.Request,
 }
 
 func pingHandler(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+	w.Header().Add("Supported-Platforms", system.SupportedPlatforms())
```

Comment:
That's set in `WrapHandler` in the middleware and does it for every request by the look of it, rather than just on `_ping`.